Sources/BraintreePayPal/BTPayPalAnalytics.swift

```diff
@@ -0,0 +1,24 @@
+import Foundation
+
+class BTPayPalAnalytics {
+    
+    // MARK: - Tokenize Events Counted in Conversion Rates
+    static let vaultRequestStarted = "paypal:vault-tokenize:started"
+    static let checkoutRequestStarted = "paypal:checkout-tokenize:started"
+    
+    static let tokenizeFailed = "paypal:tokenize:failed"
+    static let tokenizeSucceeded = "paypal:tokenize:succeeded"
+    static let browserLoginCanceled = "paypal:tokenize:browser-login:canceled"
+   
+    // MARK: - Tokenize Events Additional Detail analytic Messages
+    static let tokenizeNetworkConnectionFailed = "paypal:tokenize:network-connection:failed"
+
+    static let browserPresentationStarted = "paypal:tokenize:browser-presentation:started"
+    static let browserPresentationSucceeded = "paypal:tokenize:browser-presentation:succeeded"
+    static let browserPresentationFailed = "paypal:tokenize:browser-presentation:failed"
+    
+    static let browserLoginFailed = "paypal:tokenize:browser-login:failed"
+    // specific cancel from permisison alert
+    static let webSessionAlertCanceled = "paypal:tokenize:web-session-alert:canceled"
```

Comment:
I think we can remove the word tokenize from all of the browser-login/web-session/browser-presentation events since we will have captured the tokenize status in our other events.

Reply:
We also send general tokenizeFailed when we are returning to the app with a failure even with browser events. It's all part of the tokenize process with success being completion with BTPayPalNonce. Would it be meaningful to reflect that?

Reply:
Yep, sending the tokenize event with the browser failure is fine! We can include tokenize if we find it helpful. It looks like the [PPCP events](https://paypal.sharepoint.com/:x:/r/sites/BTSDK/_layouts/15/Doc.aspx?sourcedoc=%7BAAA7C353-5744-4F21-AC7D-DF85D8605817%7D&file=Northstar%20Analytics%20Events.xlsx&action=default&mobileredirect=true&cid=1b0afccd-68ab-448f-9760-60ed14df9811) don't pass the method name which I was comparing this too. We've already diverged from that so maybe it's fine. 🤷

Reply:
I think we just have additional detailed messages, rather than having diverged. I guess the difference is tokenize word. I was following the initial instructions for the scheme. [module: method: event: started/succeeded/failed].

Reply:
@scannillo thoughts?

Reply:
We've diverged in that we include the method name whereas we don't on PPCP. As some point it likely makes sense to align the two and either include the method on PPCP or remove it from BT, if that makes sense. We can punt on that though and leave it as is for now!

I initially thought on PPCP we did <module:method:status> for the equivalent to tokenize and <module:status> for the browser based events which is where my comment was coming from, but I was just misremembering.

Reply:
so on PPCP web payments, we have "paypal-web-payments:started". We have "paypal:tokenize:started".  I think "ios" information was going to come from metadata. I guess  our "tokenize" is taking place of "payments".

Reply:
yes, I guess it reads better -` tokenize: started` rather than "`paypal:started`"  since "ios" is in meta-data

Reply:
I'm fine removing the word "tokenize" on the browser events like I discussed with Jax privately.

Reply:
I think we should leave tokenize for now - again I was misremembering. I'll resolve this conversation.